### PR TITLE
Refine lint warning on homepage

### PR DESCRIPTION
**Pull request by saltcod**
Refine the homepage Security Issues badge: 

separate warn and error lints — link to the warn tab if only warn lints exist 

To confirm: 

1. create a foreign data wrapper using Stripe with defaults. This will generate a warn lint.
2. this will show an amber dot and link to the warn tab in Security Advisor
3. next, run the countries quickstart in the SQL Editor. This will generate an error lint. 
4. this will show a red dot and link to the error tab in the Security Advisor 

screenshot-2024-08-06-at-15 14 25

### CODE REVIEW

File: apps/studio/components/interfaces/Home/SecurityStatus.tsx [hunk 1]
```diff
@@ -1,26 +1,27 @@
...
+  const { data, isLoading } = useProjectLintsQuery({ projectRef: project?.ref })
 
-  const securityLints = (data ?? []).filter(
-    (lint) =>
-      lint.categories.includes('SECURITY') && (lint.level === 'ERROR' || lint.level === 'WARN')
-  )
+  const securityLints = (data ?? []).filter((lint) => lint.categories.includes('SECURITY'))
+  const errorLints = securityLints.filter((lint) => lint.level === 'ERROR')
+  const warnLints = securityLints.filter((lint) => lint.level === 'WARN')
+  const noIssuesFound = errorLints.length === 0 && warnLints.length === 0
```
Comment (charislam) [inline apps/studio/components/interfaces/Home/SecurityStatus.tsx:17]: question: security lints will never be `INFO` level?

**Review by charislam — APPROVED** [review]

**Comment by charislam** [comment]
Tested as documented, instructions are great 😄

File: apps/studio/components/interfaces/Home/SecurityStatus.tsx (hunk 1 above)
Comment (saltcod) [reply apps/studio/components/interfaces/Home/SecurityStatus.tsx:17]: there are info level lints, but I figure they're not high prio enough to trigger these notifications

File: apps/studio/components/interfaces/Home/SecurityStatus.tsx [hunk 2]
```diff
@@ -33,8 +34,12 @@ export const SecurityStatus = () => {
             ) : (
               <div
                 className={cn(
-                  `w-2 h-2 rounded-full`,
-                  noIssuesFound ? 'bg-brand' : 'bg-destructive-600'
+                  'w-2 h-2 rounded-full',
+                  noIssuesFound
```
[the remark this answers is not in the document]
Comment (saltcod) [reply apps/studio/components/interfaces/Home/SecurityStatus.tsx:32]: aah good catch thanks!

**Pull request merged**